### ISSUE TITLE
Template/HTML Folding not folding multi line elements properly

### ISSUE THREAD

**Issue by mika76** [vuejs/vetur#1294]
- [X] I have searched through existing issues
- [X] I have read through [docs](https://vuejs.github.io/vetur)
- [X] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: MacOs
- Vetur version: 0.20.0
- VS Code version:1.33.1

## Problem

With the following htm (where attributes have been split over multiple lines and closing bracket is on own line)

```html
<div
    class="someclass"
    style="somestyle"
>
   <div class="inner"></div>
   <div class="inner2"></div>
</div>
```

When trying to fold/collapse the first div, it does not collapse until it's closing `</div>` tag but only collapses the opening tag and it's attributes. The rest of the inner tags are still exposed...

Basically it ends up looking like:

```html
<div...
   <div class="inner"></div>
   <div class="inner2"></div>
</div>
```

In a normal HTML file all the inner tags are also folded and hidden and only the start and end tag are visible. eg:

```html
<div...
</div>
```

## Reproducible Case

Very easy to reproduce. Just split multiple attributes of a tag onto their own lines. Then try and fold that element.

**Comment by MisterY** [comment]
The closing ">" is in a separate line. If I move it up after the last attribute, the folding works as expected.
However, the next doc formatting moves it back into a separate line.

**Comment by mika76** [comment]
Yeah, prettier too. The thing is that a normal html file does not collapse like this. So all I'm saying is it should be consistent.

**Comment by octref** [comment]
Vetur now only have indentation based folding: https://github.com/vuejs/vetur/issues/899#issuecomment-435943171.

This becomes possible after #899.

**Comment by mika76** [comment]
@octref awesome - can't wait 😊

**Comment by mika76** [comment]
This is related to #2234 and was fixed in #2244
